vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data_unittest.cc

```diff
@@ -174,10 +189,14 @@ TEST_F(BatAdsVectorDataTest, AddElementWise) {
 
 TEST_F(BatAdsVectorDataTest, DivideByScalar) {
   // Arrange
-  VectorData v1 = VectorData({0.4F, 0.3F, 0.8F});
-  VectorData v2 = VectorData({1.9F, -0.75F, 0.0F});
-  VectorData v3 = VectorData({0.0F, 0.0F, 0.0F});
-  VectorData v4 = VectorData({0.8F, 0.2F, -0.35F});
+  const std::vector<float> data1 = {0.4F, 0.3F, 0.8F};
+  const std::vector<float> data2 = {1.9F, -0.75F, 0.0F};
+  const std::vector<float> data3 = {0.0F, 0.0F, 0.0F};
+  const std::vector<float> data4 = {0.8F, 0.2F, -0.35F};
+  VectorData v1 = VectorData(data1);
```

Comment:
Is this approach now required because of the multiple initialization methods?

Reply:
yes, but will revert now that we said deserialisation/serialisation should not be in VectorData